core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -19,6 +19,7 @@
 
 var forms = require('./forms.js');
 var waitFor = require('./waitFor.js');
+var workflow = require('../protractor_utils/workflow.js');
```

Comment:
Do you actually need to import this if you don't use it anywhere?

Reply:
Done.